cypress/support/commands.ts

```diff
@@ -34,7 +34,7 @@ Cypress.Commands.overwrite('tab', (originalFn, subject) => {
   return Cypress.Promise.try(() => originalFn(subject))
     .then(value => {
       log.set('$el', value).snapshot();
-      return value;
+      return Cypress.$(value);
```

Comment:
This isn't a strictly necessary change, but sometimes the value that comes back is a DOM element instead of a jQuery-wrapped element. This happens when the focus trap cancels the event. The plugin returns `document.activeElement` instead of `$(document.activeElement)`. This line normalizes the return value.